nat/src/stateful/test.rs

```diff
@@ -3,6 +3,7 @@
 
 #[cfg(test)]
 mod tests {
+    use crate::StatefulNat;
```

Comment:
Generally the "paragraphs" I use are, `std::` deps, outside crate deps, workspace deps, and the `crate::` deps or sub-modules.  Doesn't really matter, just FYI.  Not sure how this ended up the way it did.